src/bot.py

```diff
@@ -51,9 +53,12 @@ def main():
     # log all errors
     dispatcher.add_error_handler(handle_error)
 
-    # Start the Bot
+    # start the Bot
     updater.start_polling()
 
+    # start the scheduler which broadcasts the menus
+    scheduler(dispatcher.bot)
```

Comment:
Is this really all you need to do? I was under the impression there needed to be a `while(true)` loop, and a call to `run_pending()`
Also, this means that busy waiting is being used. Might probably want to have a separate thread handle the scheduler

Reply:
it seems that it works for the first iteration at the scheduled time (i set to last night)
Maybe there's a while loop needed. Just need to put the thread to sleep in between.